### PR TITLE
=Target User Profile and Value Prop

### PR DESCRIPTION
**Pull request by zlimez**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -257,13 +257,15 @@ _{Explain here how the data archiving feature will be implemented}_
...
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* An artist who works through commissions
+* has a need to manage clients and commissions
+* wants to review past commission works and feedbacks
+* wants to market himself to the right clients
+* prefers desktop apps over other types
+* prefers using GUI over CLI
```
Comment (nvmdava) [inline docs/DeveloperGuide.md:265]: For v1.2 graphic interface won't be implemented, so maybe keep this as 

- is reasonably comfortable using CLI apps

Comment (ngxingyu) [reply docs/DeveloperGuide.md:265]: We also have to adhere to the non-functional requirement for catering to users who prefer typing: https://nus-cs2103-ay2223s1.github.io/website/admin/tp-constraints.html#constraint-typing-preferred. Perhaps we can bring back the original line on preferring typing to fulfil this requirement.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -257,13 +257,15 @@ _{Explain here how the data archiving feature will be implemented}_
...
+
+**Value proposition**:
+Allows artists to hone their craft systematically and tune their artworks to fit the taste of their clients which ultimately builds their reputation.
```
Comment (ngxingyu) [inline docs/DeveloperGuide.md:268]: I see our product primarily as a way for the artists to track the customers and commissions and iterations easily, and being able to improve their skills is a byproduct of this, perhaps we can be clearer about what the app can do, and provide a clearer explanation of how the app helps the artists improve their skills.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -257,13 +257,15 @@ _{Explain here how the data archiving feature will be implemented}_
...
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* An artist who works through commissions
+* has a need to manage clients and commissions
+* wants to review past commission works and feedbacks
+* wants to market himself to the right clients
```
Comment (ngxingyu) [inline docs/DeveloperGuide.md:263]: Should we avoid using gender-specific terms?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zlimez) [reply docs/DeveloperGuide.md:265]: Ok got it

**Pull request closed without merging**